src/Backend/Spec.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Command} from './Command';
+
+/**
+ * Spec is a bundle of imformation that could be used for certain objective.
```

Comment:
imformation -> information

Reply:
I'm not sure but

```suggestion
 * Spec is a bundle of information that could be used for checking whether something to require is satisfied with a certain objective.
```

Reply:
Eg.) A and B
- B shows/has its own spec
- A see/read B's spec
- Now A can realize that A can use B or not